tests/fv3_conf/cpld_bmark_run.IN

```diff
@@ -52,9 +52,43 @@ if [[ $CPLWAV == .T. && $CPLWAV2ATM == .T. ]]; then
   fi
 fi
 
-# ICs
-cp    @[INPUTDATA_ROOT]/${FV3_IC}/sfc_data*.nc ./INPUT
-cp    @[INPUTDATA_ROOT]/${FV3_IC}/gfs_data*.nc ./INPUT
-cp    @[INPUTDATA_ROOT]/${MOM6_IC}/MOM*.nc ./INPUT
-# the BM ICs are still named cice5 and need to remain so until P5.0 is completed
-cp    @[INPUTDATA_ROOT]/${CICE_IC}/cice5_model_@[ICERES].*.nc ./cice_model.res.nc
+# No restart
+if [ $WARM_START = .F. ]; then
+  # ICs
+  cp    @[INPUTDATA_ROOT]/${FV3_IC}/sfc_data*.nc ./INPUT
+  cp    @[INPUTDATA_ROOT]/${FV3_IC}/gfs_data*.nc ./INPUT
+  cp    @[INPUTDATA_ROOT]/${MOM6_IC}/MOM*.nc ./INPUT
+  # the BM ICs are still named cice5 and need to remain so until P5.0 is completed
+  cp    @[INPUTDATA_ROOT]/${CICE_IC}/cice5_model_@[ICERES].*.nc ./cice_model.res.nc
```

Comment:
Maybe in the future we can consider changing "cice5_model" to "cice_model" to avoid confusion as we are running cice6?

Reply:
Yes, I think we do need to do this. We can change it in the RT since the files are no longer sym-linked to the BM input directory that the workflow uses. But there is potential confusion if the workflow continues to use the name cice5 and the RT uses cice6.